charts/tfy-agent/values.yaml

```diff
@@ -432,6 +432,66 @@ tfyAgentProxy:
     ## 
     name: ""
 
+  clusterRole:
+    ## @param tfyAgentProxy.clusterRole.enable Create cluster role.
+    enable: true
+    ## @param tfyAgentProxy.clusterRole.rules [array] Cluster role Authz rules.
+    rules:
+      - apiGroups: [""]
```

Comment:
Should these roles be in values or part of the manifest?

Reply:
I am keeping it in values to be easily discoverable and fixable.